package.json

```diff
@@ -66,5 +67,7 @@
     "webpack-dev-server": "^4.15.1",
     "webpack-merge": "^5.8.0"
   },
-  "sideEffects": false
+  "sideEffects": [
+    "*.css"
+  ]
```

Comment:
needed this to tell webpack about our css (tree splitting was ignoring css files because we told it we had no side effects..).

IIUC this is a webpack specific thing. see https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free